### PR TITLE
Update ALTEX

### PR DESCRIPTION
**Pull request by cparnot**
ALTEX update based on user input, see https://forums.zotero.org/discussion/55362/reporting-style-errors-for-alternatives-to-animal-experimentation-altex/#Item_0 , as well as personal communication directly from the same person.

### CODE REVIEW

**Comment by cparnot** [comment]
I recommend you wait for my feedback before merging. I am waiting on feedback from the original requestor (Sylvie).

File: alternatives-to-animal-experimentation.csl [hunk 1]
```diff
@@ -170,10 +170,9 @@
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic"/>
             <group prefix=" ">
-              <text variable="volume" suffix=": "/>
+              <text variable="volume"  font-style="italic" suffix=": "/>
```
Comment (rmzelle) [inline alternatives-to-animal-experimentation.csl]: (there's a double-space before "font-style")

**Comment by rmzelle** [comment]
@cparnot, heard back yet?

**Comment by cparnot** [comment]
@rmzelle I did not hear back yet, so I sent her another email.

**Comment by SylvieVullioud** [comment]
Hello, 
Sorry for the long time for reply. I could not find how to execute the code in Zotero. I had to subscribe to Github. And to check the modification: as you understand, I'm very bad for informatics...It took me time to be able to answer to you. First: THANK YOU for giving time like this. 

Here is what I noticed by comparison with author editor instructions
- year date of publication is in square brackets, editor asks for simple brackets.
- no need of : after journal title, which should be followed by a , directly, there is a need of automatic 
- in the case of gold-OA journal or for accepted manuscript in OA repository, there is the need of automatic generation of url permanent link. 
Ex Sauer, U.G., T. Wächter, B. Grune, et al. (2009). Go3R - semantic Internet search engine for alternative methods to animal testing. ALTEX 26 , 17–31. http://www.altex.ch/Current-issue.50.html?iid=104&aid=2.

I tried to motivate editor in chief of Altex to take contact also with CSL community to try to work together to offer author a clean usable csl Altex style. Altex is owned by Springer, running also Papers. Would it be possible also to ask publisher Springer to update entirely Altex style?

Keep in touch for any useful infos. You understood that I'm very slow, but I try to do my best to follow: be patient, thank you!

Merry Christmas,
Sylvie

**Comment by rmzelle** [comment]
@SylvieVullioud, in case you haven't already figured it out, there is some documentation on installing CSL styles in Zotero at https://www.zotero.org/support/styles#installing_additional_styles.

And, yes, Springer has been standardizing most of its journals to use one of a small set of citation formats, so you can certainly ask the Altex editor whether he's willing to consider to switch to a standard Springer citation format.

@cparnot, will you update your pull request based on @SylvieVullioud's feedback?

**Comment by cparnot** [comment]
@SylvieVullioud I am not sure what you mean here:

> no need of : after journal title, which should be followed by a , directly, there is a need of automatic

Could you clarify?

Also, you mention this:

> year date of publication is in square brackets, editor asks for simple brackets.

I have made the correction.

> in the case of gold-OA journal or for accepted manuscript in OA repository, there is the need of automatic generation of url permanent link. Ex Sauer, U.G., T. Wächter, B. Grune, et al. (2009). Go3R - semantic Internet search engine for alternative methods to animal testing. ALTEX 26 , 17–31. http://www.altex.ch/Current-issue.50.html?iid=104&aid=2.

I would expect Altex takes care of that?

**Comment by cparnot** [comment]
@rmzelle I actually addressed the other issue brought up by @SylvieVullioud. After a bit more work, I understood she meant the extra colon that was there for no good reason after the volume number.

As far as I am concerned, if the PR passes the automatic tests, it's ready to be accepted into the main branch.

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**